content/kubermatic/master/tutorials-howtos/external-clusters/eks/_index.en.md

```diff
@@ -1,32 +1,53 @@
 +++
-title = "Adding an External EKS Kubernetes Cluster"
+title = "Amazon Elastic Kubernetes Service"
 date = 2022-01-10T14:07:15+02:00
-description = "Detailed tutorial to help you add an existing Kubernetes cluster in EKS and then manage it using KKP"
+description = "Detailed tutorial to help you add an existing Kubernetes cluster in EKS"
 weight = 7
 
 +++
 
 ## Add EKS Cluster
 
 You can add an existing Kubernetes cluster and then manage it using KKP.
-From the `Clusters` page, click `External Clusters`. Click the `Add External Cluster` button and pick `Elastic Kubernetes Engine` provider.
 
-![Add External Cluster](/img/kubermatic/master/tutorials/external_clusters/add_external_cluster.png "Add External Cluster")
+- Navigate to `External Clusters` page.
 
-Select preset with valid credentials or enter EKS `Access Key ID`, `Secret Access Key` , and `Region` to connect to the provider.
+- Click the `Import External Cluster` button
+
+![External Cluster](/img/kubermatic/master/tutorials/external_clusters/external_cluster_page.png "External Cluster")
+
+- Pick `Elastic Kubernetes Engine` provider.
+
+![Select Provider](/img/kubermatic/master/tutorials/external_clusters/connect.png "Select Provider")
+
+- Provide Credentials in either of the below mentioned ways:
+    - Select a pre-created preset which stores the provider specific credentials. 
+
+      Create a preset on your KKP cluster with `spec.eks.accessKeyID` , `spec.eks.secretAccessKey` and `spec.eks.region`.
+    - Manually enter the credentials `Access Key ID`, `Secret Access Key` and select `Region`
+
+{{% notice info %}}
+Region is also kept a part of preset as its a required value to create a EKS client.
```

Comment:
`Region is also kept a part of preset as its a required value to create a EKS client.` - > `Region is also kept as a part of a preset as it is a required value to create an EKS client.`